src/index.ts

```diff
@@ -156,6 +142,10 @@ class ForkTsCheckerWebpackPlugin {
     this.vue = options.vue === true; // default false
   }
 
+  static getHooks(compiler: webpack.Compiler) {
```

Comment:
```suggestion
  static getCompilerHooks(compiler: webpack.Compiler) {
```

That would be our naming convention, but technically you are free to choose whatever you like.

We have chosen this naming to allow adding hooks for other objects later, i. e. `getCompilationHooks`.